apps/super-app/src/components/navbar/NavBar.css

```diff
@@ -0,0 +1,38 @@
+@import url('https://fonts.googleapis.com/css?family=Hind');
+a {
+  text-decoration: none;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0px 32px 10px;
+  gap: 56px;
```

Comment:
The left and right alignment looks off with the `gap`. I think we can remove it.